lib/core/src/Cardano/Wallet.hs

```diff
@@ -1764,6 +1764,7 @@ data ErrCreateRandomAddress
     = ErrIndexAlreadyExists (Index 'Hardened 'AddressK)
     | ErrCreateAddrNoSuchWallet ErrNoSuchWallet
     | ErrCreateAddrWithRootKey ErrWithRootKey
+    | ErrCreateAddressInvalidWalletType
```

Comment:
The constructor name`ErrCreateAddressInvalidWalletType` makes it seem like the error can be thrown for any wallet, but the error message contains: `Make sure to use Byron random wallet id.` — i.e. specific to Byron wallets.

A bit nit-picky, but maybe mention Byron in the name, like `ErrCreateAddressNotAByronWallet`, or take the expected wallet type name as a string.

Reply:
`ErrCreateAddressNotAByronWallet` it is.